wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -225,3 +232,39 @@ task FilterAndConcatVcfs {
         docker:"us.gcr.io/broad-dsp-lrma/lr-gcloud-samtools:0.1.2"
     }
 }
+
+
+# filter out singletons (i.e., keep MAC >= 2) and concatenate with deduplication
+task FilterPhasingInfo {
+
+    input {
+        File sv_vcf            # biallelic
+        File sv_vcf_tbi
+        String prefix
+    }
+
+    command <<<
+        set -euxo pipefail
+
+        # filter SV singletons
+        bcftools +setGT ~{sv_vcf} -- -t a -n u | bgzip > ~{prefix}.vcf.gz
+        bcftools index --tbi ~{prefix}.vcf.gz
+
+
+    >>>
+
+    output {
+        File filter_phasing_vcf = "~{prefix}.vcf.gz"
+        File filter_phasing_vcf_tbi = "~{prefix}.vcf.gz.tbi"
+    }
+    ###################
+    runtime {
+        cpu: 1
+        memory:  "4 GiB"
+        disks: "local-disk 50 HDD"
+        bootDiskSizeGb: 10
+        preemptible_tries:     3
+        max_retries:           2
+        docker:"hangsuunc/cleanvcf:v1"
```

Comment:
We should avoid personal Docker images in production code. This task simply uses bcftools as does `FilterAndConcatVcfs`, so is there no reason they cannot use the same Docker (e.g., different bcftools versions)?

Reply:
I tried "us.gcr.io/broad-dsp-lrma/lr-gcloud-samtools:0.1.2", it seems bcftools +setGT doesn't work for this version of bcftools. Should we update the public docker somehow?

Reply:
I don't think it is a version issue, since that Docker contains bcftools 1.18 and the +setGT plugin predates that. Rather, I think that bcftools in the Docker might not be set up to use *any* plugins. @SHuang-Broad might have some opinions on what to do here.